File: python/01-sequencias/main009.py
'''Faça um algoritmo que leia quanto dinheiro uma pessoa tem na carteira (em R$)
e mostre quantos dólares ela pode comprar. Considere US$1,00 = R$3,45.

Data 26/09/2025 -> 5ª Dia
'''

dinheiro = float(input('Quanto dinheiro tens carteira: '))

dolar = dinheiro / 3.45

print(f"Com R${dinheiro} posso comprar US${dolar:.2f}")

File: c/01-sequencias/main009.c
/*Faça um algoritmo que leia quanto dinheiro uma pessoa tem na carteira (em R$)
e mostre quantos dólares ela pode comprar. Considere US$1,00 = R$3,45.

Data 26/09/2025 -> 5ª Dia
*/

#include <stdio.h>

int main() {

    float dinheiro;
    float dolar;

    scanf("%f",&dinheiro);

    dolar = dinheiro / 3.45;

    printf("Com R$%f posso comprar US$%.2f",dinheiro,dolar);
    return 0;
}